### PR TITLE
Add support for multiple server permanent keys

### PR DESCRIPTION
**Pull request by lgrahl**
@dbrgn Wanna have a look?

### CODE REVIEW

File: saltyrtc/server/bin.py [hunk 1]
```diff
@@ -142,9 +143,10 @@ def generate(key_file):
 @click.option('-sk', '--sslkey', type=click.Path(exists=True), help=_h("""
 Path to a file that contains the SSL private key. Will be read from
 the SSL certificate file if not present."""))
-@click.option('-k', '--key', type=click.Path(), help=_h("""
+@click.option('-k', '--key', type=click.Path(), multiple=True, help=_h("""
```
Comment (dbrgn) [inline saltyrtc/server/bin.py]: Is the ordering of keys guaranteed to be stable when specifying multiple keys?

File: saltyrtc/server/server.py [hunk 2]
```diff
@@ -561,6 +562,42 @@ def keep_alive_loop(self):
...
+            server_key = self._server.keys.get(message.server_key)
+            if server_key is None:
+                raise ServerKeyError(
+                    'Server does not have the requested permanent public key')
+
+            # Set the key instance on the client
+            client.server_permanent_key = server_key
+        elif server_keys_count > 0:
+            # Use primary permanent key
+            client.server_permanent_key = next(iter(self._server.keys.values()))
```
Comment (dbrgn) [inline saltyrtc/server/server.py]: Why not just `self._server.keys.values()[0]`?

Comment (lgrahl) [inline saltyrtc/server/server.py]: `TypeError: 'ValuesView' object does not support indexing`

File: saltyrtc/server/bin.py (hunk 1 above)
Comment (lgrahl) [inline saltyrtc/server/bin.py]: Tested it manually beforehand. Maybe we should add a test for that... just in case.

Comment (lgrahl) [inline saltyrtc/server/bin.py]: `test_serve_asyncio_2nd_key` and `test_serve_asyncio_2nd_key_reversed` test it.

File: saltyrtc/server/server.py (hunk 2 above)
Comment (dbrgn) [inline saltyrtc/server/server.py]: Ah, I see :) Sorry.

Comment (lgrahl) [inline saltyrtc/server/server.py]: No worries

**Pull request merged**